slowlog/slowlog.go

```diff
@@ -49,22 +96,28 @@ func (s *slowLog) FetchSlowLog(size int) ([]SlowLogData, error) {
 		var log SlowLogData
 		var args []string
 		if len(entry) > 5 {
-			redis.Scan(entry,
+			_, err = redis.Scan(entry,
 				&log.Id,
 				&log.Timestamp,
 				&log.Duration,
 				&args,
 				&log.ClientAddress,
 				&log.ClientName)
 		} else {
-			redis.Scan(entry,
+			_, err = redis.Scan(entry,
 				&log.Id,
 				&log.Timestamp,
 				&log.Duration,
-				nil,
 				&args)
 		}
-
+		if err != nil {
+			fmt.Println(fmt.Sprintf("Error during redis.Scan: %v", err))
+			continue
+		}
+		if s.isIDExisting(log.Id) {
+			fmt.Println(fmt.Sprintf("Skipping SlowLog(id=%v)!", log.Id))
```

Comment:
It's that not create too much noise ?
Can you add a small logger lib. That we can define, level of logging ?

Thanks